modules/500-upmeter/templates/upmeter-agent/rbac-for-us.yaml

```diff
@@ -29,6 +29,10 @@ rules:
   - apiGroups: [""]
     resources: ["pods"]
     verbs: ["get", "list"]
+  # - In deckhouse-controller probe we need to discover deckhouse-leader service
+  - apiGroups: [""]
+    resources: ["services"]
+    verbs: ["get", "list"]
```

Comment:
As noted above, why do we need listing access? We can also add resourceNames for that specific `deckhouse-leader` service.

Reply:
changed